inspirehep/modules/literaturesuggest/forms.py

```diff
@@ -344,26 +348,34 @@ class LiteratureForm(INSPIREForm):
         export_key='abstract',
     )
 
-    languages = [("en", _("English")),
-                 ("rus", _("Russian")),
-                 ("ger", _("German")),
-                 ("fre", _("French")),
-                 ("ita", _("Italian")),
-                 ("spa", _("Spanish")),
-                 ("chi", _("Chinese")),
-                 ("por", _("Portuguese")),
-                 ("oth", _("Other"))]
+    language_choices = [
+        (language, Locale('en').languages.get(language))
+        for language in FORM_LANGUAGES
+    ]
```

Comment:
Since it's anyway a short list, shouldn't they be alphabetically sorted by Label?